os/arch/arm/src/imxrt/imxrt_clock.c

```diff
@@ -0,0 +1,1129 @@
+/****************************************************************************

```

Comment:
Please format new files as per tizenRT coding guidelines

Reply:
I fixed it
Please check it